hhs_oauth_server/settings/base.py

```diff
@@ -325,6 +326,25 @@
 EMAIL_SSL_KEYFILE = env("DJANGO_EMAIL_SSL_KEYFILE", None)
 EMAIL_SSL_CERTFILE = env("DJANGO_EMAIL_SSL_CERTFILE", None)
 
+# Get target environment (IE: DEV, TEST, IMPL, PROD)
+TARGET_ENV = env('TARGET_ENV', 'DEV')
+
+# Get aws EC2 instance metadata for logging, if available
+if bool_env(env('DJANGO_GET_EC2_METADATA', True)):
```

Comment:
Dave, just for my clarity - you are not using the ec2 metadata yet, correct?

Reply:
> Dave, just for my clarity - you are not logging the ec2 metadata yet, correct?

@oragame TY for reminding me to check. This is working. It defaults to True, but is set to False in docker-compose/web-env-vars.env when running in the local dev setup.

When testing via canary in the TEST env, this is what the firehose event JSON looks like:

{
  "time_of_event": "2021-09-14T14:13:02",
  "instance_id": "i-09ab7640b70a4c288",
  "image_id": "ami-019ff2daa7e8baac7",
  "component": "bb2.web",
  "vpc": "test",
  "log_name": "audit.global_state_metrics",
  "type": "global_state_metrics",
  "group_timestamp": "2021-09-14T14:13:01",
  "real_bene_cnt": 0,
  "synth_bene_cnt": 50,
  "global_apps_active_cnt": 17,
  "global_apps_inactive_cnt": 2,
  "global_apps_require_demographic_scopes_cnt": 17
}

Reply:
ok, 😎